src/opm/input/eclipse/Schedule/Network/NetworkKeywordHandlers.cpp

```diff
@@ -170,23 +172,46 @@ void handleNODEPROP(HandlerContext& handlerContext)
         }
 
         if (as_choke) {
-            std::string target_group = name;
-            const auto& target_item = record.getItem<ParserKeywords::NODEPROP::CHOKE_GROUP>();
+            const auto& group = handlerContext.state().groups.get(name);
+            if (group.wellgroup()) {
+                //Wells belong to a group with autochoke enabled are to be run on a common THP and should not have guide rates
+                for (const std::string& wellName : group.wells()) {
+                    auto well = handlerContext.state().wells.get(wellName);
+
+                    //Let the wells be operating on a THP Constraint
```

Comment:
Please have a space between `//` and the actual comments. It is very nitpicking while it is the general style in most of the OPM code.

Reply:
Done